server/authentication/Auth.js

```diff
@@ -1,13 +1,19 @@
 import auth0 from 'auth0-js';
-import dotenv from 'dotenv';
+// import dotenv from 'dotenv';
 
-dotenv.config();
+// dotenv.config();
 
 const port = parseInt(process.env.PORT, 10) || 3000;
 
 export default class Auth {
     userProfile;
 
+    accessToken;
+
+    idToken;
+
+    expiresAt;
+
```

Comment:
why are you creating global variables here?

Reply:
I now use Auth0 Passport in my app.js. I am not sure I need it anymore